app/templates/app/create_condition.html

```diff
@@ -43,7 +43,7 @@ <h5 class="card-title">Installment</h5>
                   <h4>Check GTS/GTF</h4>
                   <form id="superset-link" action="">
                     <label for="event-id">Event id:</label>
-                    <input id="event-id" name="event-id" class="mr-4" type="" placeholder="Event id" value="{{ installment.contract.event_id }}">
+                    <input id="event-id" name="event-id" class="mr-4" type="" placeholder="Event id">
```

Comment:
Why you define inputs with `type` attribute blank? 
Based on the provided name, you could choose for `type='number'` or just put `type='text'`(and also you can put a validator pattern in HTML)